scripts/evaluate_mesh_on_grants.py

```diff
@@ -0,0 +1,47 @@
+"""
+Evaluate MeSH model on grants annotated with MeSH
+
+The script works on an Excel file that contains 
+columns that follow the pattern below
+Disease MeSH#TAG_INDEX (ANNOTATOR)
```

Comment:
Did Kirstin do all this manually? Impressive.

Reply:
This is 2000 grants and she mapped previous annotators but 100% still impressive. Shout to @kirstinwilliamson 🥇